src/Rules/Resources/Descriptions.resx

```diff
@@ -119,299 +119,397 @@
   </resheader>
   <data name="BoundingRectangleNotValidButOffScreen" xml:space="preserve">
     <value>The BoundingRectangle property is not valid, but the element is off-screen.</value>
+    <comment>The description of an accessibility problem with an element currently under inspection. The term "BoundingRectangle" should not be translated (i.e. it should be treated as a loanword).</comment>
```

Comment:
I think I'd drop the loanword portion from all of these. I've seen non-localized words flagged along these lines: "BoundingRectangle" is a UIA property name and should not be localized